wolfcrypt/src/port/rpi_pico/README.md

```diff
@@ -0,0 +1,69 @@
+# wolfSSL Raspberry Pi Pico Acceleration
+
+wolfSSL supports RNG acceleration on the Raspberry Pi RP2040 and RP2350
+microcontrollers. Everything here assumes you are using the standard [Raspberry
+Pi Pico SDK](https://github.com/raspberrypi/pico-sdk).
+
+It has only been tested with the ARM cores of the RP2350, not the RISC-V cores.
+
+## RNG Acceleration
+
+The Pico SDK has RNG functions for both the RP2040 and RP2350. In the RP2040
+this is an optimised PRNG method, in the RP2350 it uses a built-in TRNG. The
+same API is used for both.
+
+## Comiling wolfSSL
+
+In `settings.c` you should uncomment:
```

Comment:
`settings.h`?. We typically don't recommend this approach. Recommendation is WOLFSSL_USER_SETTINGS and user_settings.h. The reason is making changes to settings.h causes issue with new software updates.